Tasks/JavaToolInstallerV0/task.json

```diff
@@ -52,7 +52,9 @@
             "required": true,
             "options": {
                 "AzureStorage": "Azure Storage",
-                "LocalDirectory": "Local Directory"
+                "LocalDirectory": "Local Directory",
+                "AptGet": "Apt-Get (Debian or Ubuntu only)",
+                "ChocoInstall": "Chocolatey (Windows only)"
```

Comment:
What do you think about just calling this 'Package Installer'?  Then use OS detection to decide on whether to use Apt-Get, ChocoInstall, (and possibly in the future Brew)?  The bonus being that the same build definition could then be used for different OSes.